ocaml/xapi/importexport.ml

```diff
@@ -340,6 +343,7 @@ module Format = struct
   let content_type = function
     | Raw -> "application/octet-stream"
     | Vhd -> "application/vhd"
+    | Tar -> "application/tar"
```

Comment:
There is no `application/tar` in `/etc/mime.types`, there is an `application/x-tar` that you could use though. If you run `file --mime foo.tar` it will say `application/x-tar`.

Reply:
Thanks, I'll use x-tar. Looks like people use all kinda mimes: https://stackoverflow.com/questions/39058006/proper-mime-media-type-for-tar-files